main.js

```diff
@@ -32,9 +32,7 @@ const repositories = core
   .map((s) => s.trim())
   .filter((x) => x !== "");
 
-const skipTokenRevoke = Boolean(
-  core.getInput("skip-token-revoke") || core.getInput("skip_token_revoke"),
-);
+const skipTokenRevoke = core.getBooleanInput("skip-token-revoke") || core.getBooleanInput("skip_token_revoke");
```

Comment:
if maintainer consider this is a breaking change, please add this patch to v2 milestone like https://github.com/actions/create-github-app-token/pull/213. ..and the PR may be conflicted with this PR.